### ISSUE TITLE
No keyboard focus on copy buttons

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7285]
The copy icons cannot be reached without using a mouse

![image](https://github.com/dart-lang/pub-dev/assets/3865266/5101fe61-13f6-4873-8c54-9b4d9c4e9138)
